### ISSUE TITLE
Audio inference fails on some platforms

### ISSUE THREAD

**Issue by whbruce** [intel/video-analytics-serving#79]
Audio inference fails to start with the following error:
```
{"levelname": "INFO", "asctime": "2021-07-30 04:22:36,484", "message": "Creating Instance of Pipeline audio_detection/environment", "module": "pipeline_manager"}
{"levelname": "ERROR", "asctime": "2021-07-30 04:22:36,511", "message": "Error on Pipeline 1: gst-core-error-quark: Could not initialize (1): /root/gst-video-analytics/gst/audio_inference_elements/base/processor.cpp(167): create_handles (): /GstPipeline:pipeline11/GvaAudioDetect:detection:\nSupported primitive descriptors list is empty for node: 143", "module": "gstreamer_pipeline"}
```
This is only seen on some platforms including systems based on these processors
* Core i5-8259U
* Core i7-7567U

This is a regression, not seen in earlier releases.

**Comment by mikhail-nikolskiy** [comment]
Duplicate of DL Streamer bug, email thread "RE: 2021.4 audio tests failing on some systems"

**Comment by whbruce** [comment]
Fixed in [v0.6.1](https://github.com/intel/video-analytics-serving/releases/tag/v0.6.1-beta).
